src/client/client.cpp

```diff
@@ -216,12 +216,6 @@ void Client::loadMods()
 		m_script->on_minimap_ready(m_minimap);
 }
 
-bool Client::checkBuiltinIntegrity()
-{
-	// TODO
-	return true;
```

Comment:
should also remove the commented-out code that calls this

Reply:
I was tied over whether to delete and it and left it due to the TODO. Deleted now